command/secrets/init/secrets_init.go

```diff
@@ -36,6 +36,27 @@ func setFlags(cmd *cobra.Command) {
 	)
 
 	cmd.MarkFlagsMutuallyExclusive(dataDirFlag, configFlag)
+
+	cmd.Flags().BoolVar(
+		&params.generatesECDSA,
+		ecdsaFlag,
+		true,
+		"the flag indicating whether new ECDSA key is created",
+	)
+
+	cmd.Flags().BoolVar(
+		&params.generatesNetwork,
+		networkFlag,
+		true,
+		"the flag indicating whether new Network key is created",
+	)
```

Comment:
Why do we need this? Don't we always need the networking key?

Reply:
I explained in the previous comment

Reply:
@zivkovicmilos ping